scheduler/simple.go

```diff
@@ -147,22 +146,6 @@ func (sched SimpleScheduler) iteratorNewUsers(ctx context.Context, timeout time.
 	return errors.WithStack(helpers.FlattenMultiError(mErr))
 }
 
-func (sched SimpleScheduler) iteratorReuseUsers(ctx context.Context, timeout time.Duration, log *logger.Log,
```

Comment:
Maybe iteratorNewUsers should change names to just iterator or iterateUsers since it now handled both cases with new users for every session as well as reusing users, depending on its input parameters.

Reply:
ah sure, good point